backend/tracim_backend/lib/core/content.py

```diff
@@ -955,9 +954,11 @@ def get_pdf_preview_path(
                 page=page_number,
                 file_ext=file_extension,
             )
+        except PageOfPreviewNotFound as exc:
+            raise exc
         except UnavailablePreviewType as exc:
             raise TracimUnavailablePreviewType() from exc
-        except UnsupportedMimeType as exc:
+        except Exception as exc:
```

Comment:
I suggest to continue to except `UnsupportedMimeType` (and raise UnavailablePreview with same message ) and except Exception (and raise UnavailablePreview with message about unkown error).

Reply:
done